src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Random should be class variable in order not to create new object of it each time method is called